src/languages/fift.js

```diff
@@ -213,7 +213,7 @@ export default function(hljs) {
           /\|\+/, /\|/, /\|_/,
 
           // Should be the last:
-          /(?<=\s)\.(?=\s)/,
```

Comment:
This won't help (much), it'll just crash later - Safari doesn't support look behind yet.

Reply:
Yes, opened this by mistake again the main repo. Was meant for an internal fix.